src/lumigo_tracer/event/trigger_parsing/__init__.py

```diff
@@ -1,37 +1,5 @@
-import re
-from typing import List, Type
+from lumigo_tracer.event.trigger_parsing.step_function_parser import StepFunctionEventTriggerParser
```

Comment:
All trigger parsers moved to the core, besides the step function which doesn't belong there